lib/trainer/test_parser.rb

```diff
@@ -22,6 +22,8 @@ def self.auto_convert(config)
       files += Dir["#{containing_dir}/Test/*.xcresult/TestSummaries.plist"]
       files += Dir["#{containing_dir}/*.xcresult/TestSummaries.plist"]
       files += Dir[containing_dir] if containing_dir.end_with?(".plist") # if it's the exact path to a plist file
+      # Xcode 11
+      files += Dir["#{containing_dir}/*.xcresult"]
```

Comment:
Do you mind adding the following globs, like for the previous Xcode versions?

```ruby
files += Dir["#{containing_dir}/**/Logs/Test/*.xcresult"]
files += Dir["#{containing_dir}/Test/*.xcresult"]
```

Reply:
You got it!